### ISSUE TITLE
No TMDB

### ISSUE THREAD

**Issue by Hartianne** [L4GSP1KE/Upload-Assistant#291]
Hello. I've this movie which doesn't have any TMDB id and so when it ask to put in the TMDB id I just press enter and that just sends a huge error and aborts the program. So how to ignore TMDB id?

**Comment by loveyoursupport** [comment]
Yeah same question, i think IMDB should ignore it if no TMDB id there.

**Comment by swannie-eire** [comment]
L4G pulls info from tmdb so without a valid tmdb entry it will fail. 

Anyone can make an entry on tmdb and it only takes a few minutes. Creating an IMDb on the other hand is a nightmare.

**Comment by KingMak3r** [comment]
> L4G pulls info from tmdb so without a valid tmdb entry it will fail.
> 
> Anyone can make an entry on tmdb and it only takes a few minutes. Creating an IMDb on the other hand is a nightmare.

As stated above, it is my understanding that the TMDB API is used to pull all of the information regarding the movie/tv show. As we do not provide it with any credentials for IMDB, it would be expected to fail if the TMDB field is left blank.
